ci-operator/step-registry/ipi/conf/ibmcloud/custom-endpoints/ipi-conf-ibmcloud-custom-endpoints-commands.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
 
 REGION="${LEASED_RESOURCE}"
 CONFIG="${SHARED_DIR}/install-config.yaml"
-DEFAULT_PRIVATE_ENDPOINTS=$(mktemp)
-cat > "${DEFAULT_PRIVATE_ENDPOINTS}" << EOF
+DEFAULT_PRIVATE_ENDPOINTS="${SHARED_DIR}/eps_default.json"
+if [[ ! -f "${DEFAULT_PRIVATE_ENDPOINTS}" ]]; then
```

Comment:
If you already saved the default endpoints in `ibmcloud-provision-vpc-endpoints-gateway`, you do not need the if block here.

Reply:
cucushift-installer-rehearse-ibmcloud-ipi-private-with-ep-cos has not use ibmcloud-provision-vpc-endpoints-gateway
ref https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58984/rehearse-58984-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.18-installer-rehearse-ibmcloud-cos/1861598454527037440

Reply:
Then I have no idea why not just append new endpoints like before. Because whatever the default endpoints json file is saved or not in `${SHARED_DIR}`, you always need to update the endpoint list in the following lines when endpoints get updated.

Reply:
I use the ${DEFAULT_PRIVATE_ENDPOINTS} file in cucushift-installer-check-ibmcloud-custom-endpoints-commands.sh